Common/Global.cs

```diff
@@ -276,6 +276,8 @@ public enum MarketDataType
     {
         /// Base market data type
         Base,
+        /// QuoteBar market data type [Bid(OHLC), Ask(OHLC) and Mid(OHLC) summary bar]
+        QuoteBar,
```

Comment:
I often find it safer to add new enumeration items to the end, otherwise we're changing existing values, `MarketDataType.TradeBar` was 1, now it's 2